tests/test_doctor_cli.py

```
@@ -0,0 +1,413 @@
+from pathlib import Path
+import sqlite3
+import subprocess
+import sys
+
+import pytest
+
+from mnemosyne.doctor import write_doctor_artifacts_atomically
+
+
+ROOT = Path(__file__).resolve().parent.parent
+FIXTURE_SECRET = "doctor-cli-fixture-secret-9f7c"  # nosec - redaction regression fixture
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "FIXTURE_SECRET"

(S105)